taxonomies/categories.txt

```diff
@@ -93462,10 +93491,6 @@ pt:Doces de figo
 sv:Fikonsylter
 wikidata:en:Q65545224
 
-<en:Jams
-en:ginger jams
-fr:Confitures de gingembre
-
```

Comment:
Is it possible to keep separated "jams" vs "preserves" for ginger?

The reason is that ginger jams in EU should have 150g of fruit per 1000g. Which is not the case for preserves. (see: https://github.com/openfoodfacts/openfoodfacts-server/pull/9707/files and https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A02001L0113-20131118)

Reply:
Ah yes, didn't see that sorry. Should then put ginger jams under ginger preserves?

Reply:
Yes, that should be fine like this.